server/enums.py

```diff
@@ -5,7 +5,19 @@
 from enum import IntEnum
 
 
+ZONE_CHECKSUMS = {
```

Comment:
Why is this duplicated with the enum?

Reply:
I probably left the enum in there and forgot to remove it